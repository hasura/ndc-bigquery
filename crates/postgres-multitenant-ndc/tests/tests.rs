mod common;
use std::env;

#[tokio::test]
async fn select_by_pk() {
    let result = common::test_query("select_by_pk").await;
    insta::assert_json_snapshot!(result);
}

#[tokio::test]
async fn select_5() {
    let result = common::test_query("select_5").await;
    insta::assert_json_snapshot!(result);
}

#[tokio::test]
async fn select_int_and_string() {
    let result = common::test_query("select_int_and_string").await;
    insta::assert_json_snapshot!(result);
}
