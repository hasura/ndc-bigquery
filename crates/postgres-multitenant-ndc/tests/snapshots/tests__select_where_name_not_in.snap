---
source: crates/postgres-multitenant-ndc/tests/tests.rs
expression: result
---
[
  {
    "rows": [
      {
        "AlbumId": {
          "value": 1
        },
        "Title": {
          "value": "For Those About To Rock We Salute You"
        }
      },
      {
        "AlbumId": {
          "value": 2
        },
        "Title": {
          "value": "Balls to the Wall"
        }
      },
      {
        "AlbumId": {
          "value": 3
        },
        "Title": {
          "value": "Restless and Wild"
        }
      },
      {
        "AlbumId": {
          "value": 4
        },
        "Title": {
          "value": "Let There Be Rock"
        }
      },
      {
        "AlbumId": {
          "value": 5
        },
        "Title": {
          "value": "Big Ones"
        }
      }
    ]
  }
]
